packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue

```diff
@@ -20,17 +19,13 @@
 import { defineComponent, PropType, ref, computed } from 'vue'
 import { useColors } from '../../../services/color-config/color-config'
 import useKeyboardOnlyFocus from '../../../composables/useKeyboardOnlyFocus'
-
-const useHover = () => {
-  const isHovered = ref(false)
-  const onMouseEnter = () => { isHovered.value = true }
-  const onMouseLeave = () => { isHovered.value = false }
-
-  return { isHovered, onMouseEnter, onMouseLeave }
-}
+import { useHover } from '../../../composables/useHover'
 
 export default defineComponent({
   name: 'VaSidebarItem',
+
+  inheritAttrs: false,
+
   props: {
     to: {
       type: [String, Object] as PropType<string | Record<string, any>>,
```

Comment:
```suggestion
      type: [String, Object] as PropType<RouteLocationRaw>,
```
Import type from 'vue-router'

Reply:
done